### DISCUSSION TITLE
Multiple push locations

### DISCUSSION THREAD

**Discussion by arvid-berndtsson** [jesseduffield/lazygit#3693] [Ideas]
Hi, I started using Lazygit months ago and truly love it.

I wondered if there is a way to [push to two different origins simultaneously](https://stackoverflow.com/questions/12961514/how-to-use-bitbucket-and-github-at-the-same-time-for-one-project).

I assume I could add and push multiple origins individually, but I am interested in pressing just P, which will then push to both origins.

This might already be possible. If so, let me know. If not, is anyone else interested in this?

**Comment by stefanhaller** [comment]
Lazygit simply calls `git push`, so if you can get it to work on the command line, it should work in lazygit too.

[This blog post](https://blog.kevinlee.io/blog/2013/03/11/git-push-to-pull-from-both-github-and-bitbucket/) describes a solution that sounds like it should work (I haven't tried it though).

**Reply by rodhash** [reply]
that's a neat trick! Instead of creating two remotes for a particular repo it's possible to use a single one with two url.. thanks for sharing!

**Reply by rodhash** [reply]
Interesting that we can fetch isolated remotes but we can't pull in the same way.. for ex created a repo on github + gitlab:

Fetching first remote (gitlab):
<img width="295" alt="image" src="https://github.com/user-attachments/assets/36242599-36e4-4b14-8372-7be45ae290c3" />

Then fetching second one (github):
<img width="286" alt="image" src="https://github.com/user-attachments/assets/f344afb3-2e62-4546-81ea-72e138c14f72" />

The Pull however can't be done like that.. whatever remote I choose the pull will always run on "origin".

**Comment by arvid-berndtsson** [comment]
It looks like it. Thank you so much @stefanhaller! 😃
